daemon/kill.go

```diff
@@ -105,6 +115,9 @@ func (daemon *Daemon) Kill(container *container.Container) error {
 		return errNotRunning{container.ID}
 	}
 
+	// FIXME(vdemeester) should we disable restartpolicy for SIGKILL or not ?
```

Comment:
@mlaventure hum… true, so `SIGKILL` would be treated exactly as the other signals then. `docker stop` sends the `stop-signal` if present, and thus, the restart manager would disable the restart policy, even if the `SIGKILL` is sent afterwards.

@crosbymichael @LK4D4 @cpuguy83 @thaJeztah WDYT ? I can update to reflect that.